### ISSUE TITLE
Creation of "Semidominant mode of inheritance"

### ISSUE THREAD

**Issue by courtneythaxton** [obophenotype/human-phenotype-ontology#4239]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Semidominant mode of inheritance

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

 A mode of inheritance that is observed for traits related to a gene encoded on chromosomes in which a trait can manifest in the heterozygotes and homozygotes, with differing phenotype severity present dependent on the number of alleles affected.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
HP:0000005

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)

A common disease that would all under this inheritance pattern is Familial Hypercholesterolemia (MIM# 143890)

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
ClinGen:courtneythaxton @courtneythaxton
ClinGen:erinriggs @ErinRiggs
ClinGen:mattwright @mwright
ClinGen jimmyzqhen @jimmyzhen
ClinGen:jennygoldstein @jennygoldstein
ClinGen:marinadistefano @marinadistefano
ClinGen:tristannelson @tnavatar
ClinGen:scottgoehringer @sgoehringer

Also looping in @nicolevasilevsky as she has helped with ontologies for ClinGen.

**Comment by pnrobinson** [comment]
Adding term as HP:0032113

**Comment by courtneythaxton** [comment]
HI @pnrobinson , 

I wanted to check back in with you to see when you may push the semi dominant MOI, HP:0032113? 

thanks,
@courtneythaxton

**Comment by pnrobinson** [comment]
@drseb can we do a release?

**Comment by drseb** [comment]
yes. next week.

**Comment by courtneythaxton** [comment]
HI All, 

Thank you for working on this. I just wanted to check that this new term has gone live. 

@courtneythaxton

**Comment by nicolevasilevsky** [comment]
@pnrobinson I see the ontology has been released, but I don't see this term on the HPO website or OLS for some reason.
